dash-renderer/src/components/error/ComponentErrorBoundary.react.js

```diff
@@ -13,9 +12,14 @@ class UnconnectedComponentErrorBoundary extends Component {
             myID: props.componentId,
             myUID: uniqid(),
             oldChildren: null,
+            hasError: false,
         };
     }
 
+    static getDerivedStateFromError(_) {
+        return {hasError: true};
+    }
```

Comment:
Not finding code calling this function

Reply:
https://reactjs.org/docs/react-component.html#static-getderivedstatefromerror